geometry_msgs/PoseStamped next_waypoint
---
bool success
string message        
